packages/mermaid/src/dagre-wrapper/shapes/util.js

```diff
@@ -82,8 +85,15 @@ export const labelHelper = async (parent, node, _classes, isNode) => {
                   img.style.width = '100%';
                 }
                 res(img);
-              })
-            )
+              }
+              setTimeout(() => {
```

Comment:
Works well on chrome without this, Electron seems to be a bit different

Reply:
What about firefox?

Reply:
Worked fine too, it was really only Electron in test.